Makefile

```diff
@@ -43,6 +53,14 @@ check:
 		sort | uniq > /tmp/compat-links
 	@ (diff /tmp/compat-defs /tmp/compat-defs)
 
+install: $(addprefix install-,compat-runtime-el)
+
+install-compat-runtime-el: $(RUNTIME) $(RUNTIME:.elc=.el)
+
+install-%-el:
+	$(if $<, install -m755 -d $(DESTDIR)$(ELDIR))
+	$(if $<, install -m644 $^ $(DESTDIR)$(ELDIR))
```

Comment:
Wouldn't you also want to install the info file?

Reply:
Makes sense I forgot.